Objective-C/CBLReplicator.mm

```diff
@@ -350,7 +360,7 @@ - (void) stop {
 - (void) _stop {
     [self cancelRetry];
     
-    if (_repl) {
+    if (_repl && (_rawStatus.level != kC4Connecting || _rawStatus.level != kC4Offline)) {
```

Comment:
Can you explain about the condition here? I couldn't follow.

Reply:
i take it back, this won't happen. Since `[replicator stop]` function will not allow it to pass. Since when creating a new c4replicator, without setting the `_state`, it defaults to `kCBLStateStopped`.